02-installing-charts/README.md

```diff
@@ -260,7 +265,7 @@ There is also a shortcut that we will use here to rollback to the last release:
 
 ```console
 $ helm rollback <release_name> 0
-Rollback was a success! Happy Helming!
```

Comment:
Nit: This is the actual output from Helm, so I probably want to keep it.

Reply:
Weirdly, I was also going with "changing it to the actual output I see" and I think I have the most recent everything. Do you wanna re-check with the versions currently in play? I wonder where the disconnect is. (I'm on a Mac if that helps.)

Reply:
I think it is a change in 2.14 then! Good catch